api/src/test/java/uk/gov/dhsc/htbhf/claimant/ClaimantLifecycleIntegrationTests.java

```diff
@@ -323,6 +356,19 @@ private LocalDate progressThroughPaymentCyclesForPregnancy(LocalDate expectedDel
         return expectedDeliveryDate.minusDays((long) CYCLE_DURATION * numCycles);
     }
 
+    private LocalDate progressThroughPaymentCyclesForPregnancyWithEligibilityOverride(LocalDate expectedDeliveryDate, UUID claimId, int numCycles)
```

Comment:
I don't see what EligibilityOverride has to do with this method, am I missing something?

Reply:
The point is that the claim has an EligibilityOverride, so we are not mocking the eligibility service

Reply:
Ok, I see, in which case should this have a verifyNoInteractions for that mock?